src/components/Layout/Navbar.tsx

```diff
@@ -235,7 +140,15 @@ export const Navbar = () => {
                 </NavigationMenuContent>
               </NavigationMenuItem>
 
-              <NavigationMenuItem>{isLoggedInMvx && <BitzDropdown />}</NavigationMenuItem>
+              <NavigationMenuItem>
+                {isLoggedInMvx && (
+                  <BitzDropdown
+                    handlePlayActionBtn={() => {
+                      setShowPlayBitzModel(true);
```

Comment:
Typo? Should be "Modal" not "Model". But don't bother fixing this, I fixed it already on my branch.

Just wanted to leave as a comment as I saw it in more palces